scripts/release.ts

```diff
@@ -86,11 +109,32 @@ async function bumpVersions() {
     console.log(c.yellow('No version changes made. Exiting...'));
     process.exit(1);
   }
-  console.log(c.green('Bumped package versions.'));
+  console.log(c.green('Bumped package versions. Bumping dependencies next...'));
+
+  getPackages().forEach((pack) => {
+    const [packageName] = Object.keys(pack);
+    const [packageLocation] = Object.values(pack);
```

Comment:
There seems to be some unnecessary complexity here in getting these values from the `getPackages()` function to here. Can this be simplified?

Is the purpose of `getPackages` to get all packages that have `@cmsgov/design-system` as a dependency? If that's the case, could the name be more specific? Could more be done in that function? Less? The boundary of the responsibilities between this inlined `forEach` and that function aren't entirely clear to me.

Reply:
I agree with Patrick about the complexity here. Perhaps moving the `forEach` logic into its own function (something like `updatePackageVersion`) could make the intention more explicit? I also like the idea of combining the logic in `getPackages` and the `forEach` loop into one function to streamline the processing.

Reply:
@tamara-corbalt @pwolfert I tried my darnedest. Please let me know what questions and ideas you have after the refactor.